### ISSUE TITLE
Commit 34445bee breaks the role.

### ISSUE THREAD

**Issue by L-P** [rvm/rvm1-ansible#74]
```
TASK: [rvm_io.rvm1-ruby | Symlink ruby related binaries on the system path] *** 
failed: [example.com] => (item=bundle) => {"failed": true, "item": "bundle", "path": "/usr/local/bin/bundle", "src": "/usr/local/rvm/wrappers/default/bundle", "state": "absent"}
msg: src file does not exist, use "force=yes" if you really want to create the link: /usr/local/rvm/wrappers/default/bundle
failed: [example.com] => (item=bundler) => {"failed": true, "item": "bundler", "path": "/usr/local/bin/bundler", "src": "/usr/local/rvm/wrappers/default/bundler", "state": "absent"}
msg: src file does not exist, use "force=yes" if you really want to create the link: /usr/local/rvm/wrappers/default/bundler
changed: [example.com] => (item=erb)
changed: [example.com] => (item=executable-hooks-uninstaller)
changed: [example.com] => (item=gem)
changed: [example.com] => (item=irb)
changed: [example.com] => (item=rake)
changed: [example.com] => (item=rdoc)
changed: [example.com] => (item=ri)
changed: [example.com] => (item=ruby)
changed: [example.com] => (item=testrb)

FATAL: all hosts have already failed -- aborting
```

Only defined var is `rvm1_rubies: ["ruby-2.2.3"]`, target machine is
running Ubuntu 14.04.3 (Vagrant ubuntu/trusty64), the host is using
Ansible 1.9.2.

**Comment by L-P** [comment]
Probably fixed in 2615824a7cd50b820d9da907b7ae690a0f28aa29. Works again.

**Comment by nickjj** [comment]
Thanks for the confirmation.
